makefile

```diff
@@ -0,0 +1,13 @@
+include .env
+export
+
+dev:
+	go run main.go
+
+migrate:
+	migrate -database ${DATABASE_URL} -path migrations down
```

Comment:
Do you want to do this every time? It will delete all the data.

Reply:
I just put this here so that I don't have to memorise the command. I'm very poor at memorising things.